uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -333,6 +333,26 @@ public void connect(String clientDeviceId, String componentId, String brokerId,
         throw lastException;
     }
 
+    /**
+     * Creates MQTT connection.
```

Comment:
Update the description of method

Reply:
Description was updated in a39a3bb388a83630e9a3cf4c44a52ab0c2c8026e